plugins/bundle/plugin_test.go

```diff
@@ -262,6 +266,10 @@ func TestPluginOneShotDeltaBundle(t *testing.T) {
 
 	ensurePluginState(t, plugin, plugins.StateOK)
 
+	if plugin.status[bundleName].Type != "delta" {
+		t.Fatal("expected delta bundle")
+	}
+
```

Comment:
Can we update the checks to something like this ?

```suggestion
	status, ok := plugin.status[bundleName]
	if !ok {
		t.Fatalf("Expected to find status for %s, found nil", bundleName)
	}
	if status.Type != DeltaBundleType {
		t.Fatalf("expected delta bundle but got %v", status.Type)
	}

```

Reply:
Yep, good call. Done.